src/components/modal/modal.js

```diff
@@ -112,6 +112,7 @@ class Modal extends React.Component {
      * @type {Boolean}
      */
     this.listening = false;
+    this.state = { state: 'default' };
```

Comment:
should this be `open` or `closed`? not sure what benefit you get from `default`?

Reply:
We have used idle and default in other locations, it felt the correct thing to do, to initialise with default. Happy to set this to default as a closed if it makes more sense.